### ISSUE TITLE
Without Lang Prefix

### ISSUE THREAD

**Issue by KVRA** [hozana/next-translate-routes#20]
The option :
withoutLangPrefix

has no documentation
How can I add this ?

**Comment by zergu** [comment]
We have same issue with unintended adding locale to url path. We use language subdomains so we end up with broken urls.

**Comment by cvolant** [comment]
Subdomains... I didn't thought about that...
Yes, it is not supported yet but should be. 
I will post here when I start working on it, but I don't know yet when that will be possible. Feel free to work on it yourself : pull requests are welcome.

**Comment by cvolant** [comment]
I gave it a try [here](https://github.com/hozana/next-translate-routes/pull/65), but there is still some issues to tackle.

I tried to set localhost:3000 and localhost:3001 as different domains, then launch on instance on each port to test it locally, but even without next-translate-routes, it creates looping redirections. Do you have an idea why?

```js
    domains: [
      {
        domain: 'localhost:3000',
        defaultLocale: 'en-US',
        http: true,
      },
      {
        domain: 'localhost:3001',
        defaultLocale: 'fr',
        http: true,
      },
    ],
```

I leave it as is for now, since I need to work on other subjects. Feel free to give it a try too, you can reuse my PR. I will post here again when I will work on it again.

**Comment by KVRA** [comment]
I have created a fork to make this, I only had to change one line in fileUrlToUrl
https://github.com/KVRA/next-translate-routes/commit/4ef06b1e827a6e101b83de90a54ad6b213b68e02

But It should change not depenging on the env variable but instead based on the next config domains array, still works for me, currently using it on a big site :) 

Thanks for your job, where can I buy you a coffee ? Thanks

**Comment by cvolant** [comment]
>  I only had to change one line in fileUrlToUrl

Is it really enough?! :open_mouth:
I am quite surprised because fileUrlToUrl is not the only place where a lang prefix is added...
Do you have a way to test multi domain routing locally?

> Where can I buy you a coffee ?

:smile:
Well, if you really want to, you can do it here: https://don.hozana.org/je-donne

**Comment by KVRA** [comment]
Yes, it works and its working in prod for a while now, I use this functions to sitemap, alternative lang links etc...
Just added dev.site.com and dev.site.de to my /etc/hosts pointing to 127.0.0.1 and added those domains to next.config

I didn't made a PR since I only E2E test without unit test :P but it works ! 

If you want to see in prod the English version is gindumac.com

**Comment by cvolant** [comment]
Ok, cool.
It would be good to make something clean on the repo so that it works for everybody, but I need to test multi domain routing locally, and it did not work when I tried. Lately I have too much to do to be able to work on this (and we don't need it for ourselves) : so I need help to tackle this.
Do you have a working config I could build on? I tried [this](https://github.com/hozana/next-translate-routes/pull/65/files#diff-00f0c77b21d08471540fa380f9f8a8d49983d0db7d55beb86577c72e33bad2c4R8-R20) but even without next-translate-routes, it does not work.
